pypif_sdk/interop/mdf.py

```diff
@@ -0,0 +1,229 @@
+from pypif_sdk.readview import ReadView
+from pypif.obj.common import Value, ProcessStep
+from citrination_client import PifQuery, DatasetQuery, SystemQuery, Filter
+from ..util.citrination import get_client
+from pypif.pif import dumps
+
+import re
+
+
+def query_to_mdf_records(query=None, dataset_id=None, acl=None):
```

Comment:
Can you make this slightly more specific and call it `mdf_acl` so that it is clear what it applies to.